pkgs/data/misc/dns-root-data/default.nix

```diff
@@ -3,7 +3,13 @@
 let
 
   rootHints = fetchurl {
-    url = "https://www.internic.net/domain/named.root";
+    # Original source https://www.internic.net/domain/named.root
+    # occasionally suffers from pointless hash changes,
```

Comment:
Nit: they're not pointless, it receives updates.

Reply:
The last one was pointless: https://github.com/NixOS/nixpkgs/pull/31855#issuecomment-345683448

Reply:
And there was another on Oct 23, as discussed on https://lists.dns-oarc.net/mailman/listinfo/dns-operations  DNS will also work perfectly OK if a few root IPs are incorrect, so even in cases where real changes happen, some people may choose to remain on the old version to avoid rebuilds.